server/public/model/client4.go

```diff
@@ -580,6 +580,10 @@ func (c *Client4) permissionsRoute() string {
 	return "/permissions"
 }
 
+func (c *Client4) limitsRoute() string {
```

Comment:
These were incorrectly removed in https://github.com/mattermost/mattermost/pull/26853